Update extractor.bat

JustZerooo opened the pull request:
More informations, nicer design, a bit updated :)

<!--- (**********************************)
      (** Fill in the following fields **)
      (**********************************) --->

**Changes proposed:**

-  
-  
-  

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)


**Tests performed:** (Does it build, tested in-game, etc.)


**Known issues and TODO list:** (add/remove lines as needed)

- [ ] 
- [ ] 


<!--- Notes
- Enable the setting "[√] Allow edits from maintainers." when creating your pull request.
- If this PR only contains SQL files, open a new issue instead and post or link the SQL in the issue.
- When adding new SQL files, name them 9999_99_99_99_db_name.sql to reduce the chance of possible merge conflicts.
--->

Commit:
Update extractor.bat

More informations, nicer design, a bit updated :)

Shauren commented:
20 hours? What kind of potato machine is that

Setraxis commented:
Some Chinese oder India Computer :D Thats a random value, i saw someoe who tells that :O

Rushor commented:
Potatoe Machines ftw!

Commit:
Update extractor.bat

Remove bullshit Copy * from Buildings into vmaps. That dies vmap4assembler.exe ...

Aokromes commented:
some people may want to simply extract basic files to test server instead all.

Setraxis commented:
Fine for these peoples :)

Commit:
Update extractor.bat

Thanks to @MaxtorCoder to correct me. There is no Account Type BFA in Config.wtf anymore!

Jinnaix commented:
No, not fine for these people, your script change doesnt cover that case.

In addition, i think its misleading telling this : 
``` 
ECHO       Before Continuing, PLEASE check to ensure your 
ECHO.
ECHO          Config.wtf is set at accounttype "BFA"
ECHO.
ECHO     (They should be located in: "\World of Warcraft\_retail_\WTF\")
```
The extractors are used for WotLK too, i would say the majority use it for that. Telling something about account type will confuse people.

Setraxis commented:
Check last change. Already changed ;)

illfated commented on contrib/extractor.bat:
```diff
@@ -1,57 +1,76 @@
+ECHO.
+ECHO                             ***NOTE*** 
+ECHO This entire process will take anywhere from 2 hours to 8 hours or MORE, 
+ECHO                depending upon the speed of your machine. (If you have a potato machine *Shauren says that*)
```
```suggestion
ECHO depending upon the speed of your machine.
ECHO  (if you have a potato machine. *Shauren says that*)
```
You want to keep info text lines at max. 80 char. length, if end user has default size cmd window.

Setraxis replied:
i use default size and all fine, no change needed

Jinnaix commented:
Again - and @Aokromes already pointed that too - this whole change deletes the option to extract something specific :

```
SET /P M=Type 1, 2, 3, 4 or 5 then press ENTER:
IF %M%==1 GOTO MAPS
IF %M%==2 GOTO VMAPS
IF %M%==3 GOTO MMAPS
IF %M%==4 GOTO ALL
```

with the change in this pr you can only extract everything and this is a bad change only to make a console window look more "pretty".

illfated commented:
Thanks, I was too tired to remember that you pointed that out. I will have a further look into that sub-routine call.

illfated commented on contrib/extractor.bat:
```diff
@@ -1,57 +1,76 @@
-SET /P M=Type 1, 2, 3, 4 or 5 then press ENTER:
-IF %M%==1 GOTO MAPS
-IF %M%==2 GOTO VMAPS
-IF %M%==3 GOTO MMAPS
```
Add these features back for people who don't want to extract mmaps every time they use this batch script.
My slow 2012 computer uses more than 6 hours to extract those for WotLK, 8-12 hours for Warlords/Legion.

Commit:
Update extractor.bat

Hopefully now all you wishes are in it.

Setraxis commented:
![NgCduzJ](https://user-images.githubusercontent.com/10685816/70323405-f79b5000-182c-11ea-8e5f-01679abd9882.png)

Now all you need ...

Jinnaix commented:
Script looks good for me now.

One more thing :

The following was written for a reason because people came here with reporting this as a problem/bug all the time :

```
ECHO The vmaps extractor output text below is intended and not an error:
ECHO ..........................................
ECHO Extracting World\Wmo\Band\Final_Stage.wmo
ECHO No such file.
ECHO Couldn't open RootWmo!!!
ECHO Done!
ECHO ..........................................
ECHO.
```

I wouldn't recommend removing it but it might influence your goal of making the extractor more pretty

Setraxis commented:
Has nothing todo wit my bat file. open a new ticket for this issue. this has 0 todo with the bat file.

Commit:
Update extractor.bat

Small fix for vMaps only ...
